### ISSUE TITLE
prefer-const produces invalid autofix when using destructuring with multiple declarators

### ISSUE THREAD

**Issue by not-an-aardvark** [eslint/eslint#11699]
**Tell us about your environment**

* **ESLint Version:** master
* **Node Version:** 12.1.0
* **npm Version:** 6.9.0

**What parser (default, Babel-ESLint, etc.) are you using?**

default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```yml
rules:
  prefer-const: error
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
let {a, b} = c, d;
```

```bash
eslint --fix
```

**What did you expect to happen?**

I expected ESLint to not create a syntax error.

**What actually happened? Please include the actual, raw output from ESLint.**

ESLint produced the following autofixed code, which has a syntax error:

```js
const {a, b} = c, d;
```

[Demo link](https://eslint.org/demo/#eyJ0ZXh0IjoiLyogZXNsaW50IHByZWZlci1jb25zdDogZXJyb3IgKi9cblxubGV0IHthLCBifSA9IGMsIGQ7Iiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjo5LCJzb3VyY2VUeXBlIjoic2NyaXB0IiwiZWNtYUZlYXR1cmVzIjp7fX0sInJ1bGVzIjp7fSwiZW52Ijp7fX19)

This was discovered when running the fuzzer locally.

**Are you willing to submit a pull request to fix this bug?**

Yes, but I'm not sure when I'll have time.

**Comment by g-plane** [comment]
Should we disable autofix at this case or split it into two statements?

**Comment by mysticatea** [comment]
I think that disabling autofix is good enough.

**Comment by mdjermanovic** [comment]
Similar bugs:

```js
let {a, b, c} = {}, e, f;
```

```js
let {a, b} = {}, c = 0;
c = 1;
```
